gdal/doc/gdal_tutorial.dox

```diff
@@ -57,11 +57,10 @@ int main()
 
 In Python:
 \code
-    import gdal
-    from gdalconst import *
+    from osgeo import gdal, gdalconst
 
-    dataset = gdal.Open( filename, GA_ReadOnly )
-    if dataset is None:
+    dataset = gdal.Open(filename, gdalconst.GA_ReadOnly)
```

Comment:
Yes, please add a new commit into this branch. I'll get the global diff before applying to SVN